spanner/google/cloud/spanner/database.py

```diff
@@ -389,8 +389,7 @@ def batch(self):
         """
         return BatchCheckout(self)
 
-    def snapshot(self, read_timestamp=None, min_read_timestamp=None,
-                 max_staleness=None, exact_staleness=None):
+    def snapshot(self, **kw):
```

Comment:
Folks, enough with the bikeshedding on this. It is not important enough to justify the amount of ink being spilled on it.